[DOCS][ResponseOps][8.x] Clarify untracked alerts (#232790)
elastic/kibana: docs/user/alerting/view-alerts.asciidoc

Once a flapping alert is recovered, it cannot be changed to flapping again. Only new alerts with repeated status changes are candidates for the flapping status. 

`untracked`::

> The rule is disabled, or you've marked the alert as untracked. To mark the alert as untracked, go to the **Alerts** table, click the actions icon (…) to expand the **More actions** menu, and click **Mark as untracked**. When an alert is marked as untracked, actions are no longer generated and the alert's status can no longer be changed. You can choose to move active alerts to this state when you disable or delete rules.

Reviewer: Not mandatory, but you can shorten a bit like this:
```suggestion
The rule is disabled, or you've marked the alert as untracked. To mark the alert as untracked, go to the **Alerts** table, click the actions icon (…) to expand the **More actions** menu, and click **Mark as untracked**. When an alert is marked as untracked, its status is no longer updated and actions are no longer generated. You can choose to move active alerts to this state when you disable or delete rules.
```
